datasets/tr/capital_markets_banned_list/tr_capital_markets_banned_list.yaml

```
@@ -0,0 +1,39 @@
+title: Turkish Capital Markets Board Banned List
```

We usually write the country name proper noun (Türkiye) rather than the adjective (Turkish) 

```suggestion
title: Türkiye Capital Markets Board Banned List
```